Deployment/Mirantis-Guidance.md

```diff
@@ -26,16 +26,24 @@ Customer is using Windows Server image 2019 with Containers
 
 ## Migration risk decision guide
 
+The possible options for the migration are combinations of the following controls:
```

Comment:
added this because the guide goes straight into risks without first presenting _what_ is the migration, or how it is supposed to run. The reader needs first to understand their options.